melt_czochralski_Dadzis/multilog_eln.archive.yaml

```diff
@@ -101,31 +101,23 @@ definitions:
           section:
             m_annotations:
               eln:
-                # lane_width: 200px
+                lane_width: 400px
             quantities:
-              DAQ_6510_sensors_locations:
-                type: str
-                description: Picture of sensor locations
-                m_annotations:
-                  browser:
-                    adaptor: RawFileAdaptor
-                  eln:
-                    component: FileEditQuantity
-              DAQ_6510_sensors_list:
+              DAQ_6510:
```

Comment:
The type here is pointing to a section "type: ../upload/raw/DAQ-6510.archive.yaml#Sensors_list"

I'm not sure how the section (a class, say to speak) is looking when being used as the type of a quantity (typical types here are stuff like int, float, datetime). So I will merge this PR and then make an upload to see how it looks like

Reply:
I tried it, it looked good. I think it was similar before. But feel free to change it from quantity to section!

Reply:
So what you see is a whole Sensors_list section into the DAQ_6510 quantity? 
I will try it on my local anyway, thanks!